### PR TITLE
TRACING-3926 | Release notes for RHOSDT 3.1

### PR DESCRIPTION
**Pull request by pavolloffay**
Version(s):

4.12, 4.13, 4.14, 4.15, 4.16

Issue:

https://issues.redhat.com/browse/TRACING-3926

Link to docs preview:

https://71235--ocpdocs-pr.netlify.app/openshift-enterprise/latest/otel/otel-release-notes#otel-3-1-rn
https://71235--ocpdocs-pr.netlify.app/openshift-enterprise/latest/distr_tracing/distr_tracing_rn/distr-tracing-rn-3-1

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: otel/otel-release-notes.adoc [hunk 1]
```diff
@@ -29,6 +29,33 @@ Data enrichment and processing:: Before forwarding data to analysis tools, the C
...
+== {OTELName} 3.1
+
+{OTELName} 3.1 is based on link:https://opentelemetry.io/[OpenTelemetry] 0.93.0.
+
+[id="new-features-and-enhancements_otel-3-1-rn"]
+=== New features and enhancements
+
+This update introduces the following enhancements:
+
+* Enable target allocator in the OpenTelemetry collector.
```
Comment (max-cx) [inline otel/otel-release-notes.adoc:42]: I'll suggest a short summary of what the target allocator is and what it does (https://opentelemetry.io/docs/kubernetes/operator/target-allocator/).

File: distr_tracing/distr_tracing_rn/distr-tracing-rn-3-1.adoc [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+[options="header"]
+|===
+|Operator |Component |Version
+|{JaegerName}
+|Jaeger
+|1.53.0
+
+|xref:../../otel/otel-release-notes.adoc[{OTELName}]
+|OpenTelemetry
+|0.92.0
```
Comment (max-cx) [inline distr_tracing/distr_tracing_rn/distr-tracing-rn-3-1.adoc:23]: It's `0.93.0` in the other file:
```suggestion
|0.93.0
```

File: otel/otel-release-notes.adoc [hunk 3]
```diff
@@ -29,6 +29,33 @@ Data enrichment and processing:: Before forwarding data to analysis tools, the C
...
+
+* Nothing
+
+[id="known-issues_otel-3-1-rn"]
+=== Known issues
+
+* Nothing
+
 [id="otel-3-0-rn"]
 == {OTELName} {DTProductVersion}
```
Comment (pavolloffay) [inline otel/otel-release-notes.adoc:60]: @max-cx note that once you change DTProductVersion to the 0.93.0 the RN for 3.0 will need to be changed to accommodate the change.

File: otel/otel-release-notes.adoc [hunk 4]
```diff
@@ -29,6 +29,33 @@ Data enrichment and processing:: Before forwarding data to analysis tools, the C
...
+== {OTELName} 3.1
+
+{OTELName} 3.1 is based on link:https://opentelemetry.io/[OpenTelemetry] 0.93.0.
+
+[id="new-features-and-enhancements_otel-3-1-rn"]
+=== New features and enhancements
+
+This update introduces the following enhancements:
+
+* Add support for target allocator in the OpenTelemetry collector. The target allocator is an optional component of the operator that shards Prometheus receiver scrape targets across deployed fleet of OpenTelemetry collector. The target allocator as well provides integration with Prometheus `PodMonitor` and `ServiceMonitor` custom resources.
```
Comment (max-cx) [inline otel/otel-release-notes.adoc:42]:
```suggestion
* Add support for target allocator in the OpenTelemetry collector. The target allocator is an optional component of the OpenTelemetry Operator that shards Prometheus receiver scrape targets across the deployed fleet of OpenTelemetry Collector instances. The target allocator provides integration with the Prometheus `PodMonitor` and `ServiceMonitor` custom resources.
```

File: distr_tracing/distr_tracing_rn/distr-tracing-rn-3-1.adoc [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+This update introduces the following enhancements for the {JaegerShortName}:
+
+* Nothing
+
+[id="bug-fixes_jaeger-release-notes_distributed-tracing-rn-3-1"]
+=== Bug fixes
+
+This update introduces the following bug fixes for the {JaegerShortName}:
+
+* Fix nondeterministic jaeger-agent injection. Now the operator prefers Jaeger instance from the same namespace as the target deployment. (link:https://issues.redhat.com/browse/TRACING-3722[TRACING-3722])
```
Comment (max-cx) [inline distr_tracing/distr_tracing_rn/distr-tracing-rn-3-1.adoc:53]:
```suggestion
* Fixed nondeterministic `jaeger-agent` injection. With this update, the operator prioritizes the Jaeger instance from the same namespace as the target deployment. (link:https://issues.redhat.com/browse/TRACING-3722[TRACING-3722])
```

**Comment by max-cx** [comment]
@pavolloffay, thank you, feel free to close this Draft PR. I have transferred all the content that I needed into my RN branch.

**Pull request closed without merging**